danfojs/src/core/frame.js

```diff
@@ -58,6 +58,26 @@ export class DataFrame extends Ndframe {
             })
         });
     }
+    /**
+     * Write a CSV file of the DataFrame contents
+     * @param {string} csvFilePath Path to save CSV when in Node.js form
+     * @returns {Promise<String>} CSV representation of Object data
+     *     
+     */
+    async to_csv(csvFilePath = "") {
+        const csvContent = await super.to_csv();
+        // behave differently for Node vs Web
+        if (typeof window === "undefined") {
+            // Write CSV on Node.js
+            const fs = require("fs");
+            fs.writeFileSync(csvFilePath, csvContent, (err) => err && console.error(err));
```

Comment:
Technically this could be an `await fs.writeFile(...` ... not sure if it matters here, though!

Reply:
I wasn't sure, bc it has a callback over a promise.